src/Apps/Artwork/Components/OtherAuctions.tsx

```diff
@@ -36,3 +39,22 @@ export const OtherAuctionsFragmentContainer = createFragmentContainer(
     }
   `
 )
+
+export const OtherAuctionsQueryRenderer = () => {
```

Comment:
So to confirm, we're not going to SSR render this?

Reply:
Not now, that can be a follow up if we want it.

It's currently client-side rendered so this isn't any different 😄.